src/test/resources/environment.yaml

```diff
@@ -11,49 +12,135 @@ proxy-config:
 
 # The name of the environment
 environment-name: demo
-# The region the environment will run in
-region: us-west-2
-# Costcenter for where to bill provisioned resources. Will be tagged on all resources.
-cost-center: 11111
-# The e-mail for who owns the provisioned resources. Will be tagged on all resources.
-owner-email: obvisouly.fake@nike.com
-# The owning group for the provision resources. Will be tagged on all resources.
-owner-group: engineering team name
+
+# The primary region the environment will run in, this is where the ASG and all region specific resources will be created.
+primary-region: us-west-2
+
+# Optional map of tags that will be applied to any resource created by the CloudFormation Service in this CLI
+global-tags:
+  # Name: cerberus-{env} is automatically added to this map
+  cost-center: 11111
+  owner-email: obvisouly.fake@nike.com
+  owner-group: engineering-team-name
 
 # A IAM role ARN that will be given elevated privileges for the KMS CMK created.,
-# If you don't separate root access from admins just use the root role here
-admin-role-arn: arn:aws:iam::111111111:role/onelogin-roles-OneLoginAdminRole-2222222222
-# The Route 53 hosted zone name that will be created for CNAME records used by internal ELBs.
-vpc-hosted-zone-name: demo.internal.cerberus-oss.io
-# The hostname that will be exposed for Cerberus
-hostname: demo.cerberis-oss.io
-# The hostedZoneId that will allow the registration of the hostname
+# If you don't separate root access from admins just use the root role here.
+admin-role-arn: arn:aws:iam::111111111:role/admin
+
+# This is used to create dns records and certificates
```

Comment:
how about "This is used to create DNS records and TLS certificates"

Reply:
- [ ] Done.